### DISCUSSION TITLE
Control the transparency of a shape

### DISCUSSION THREAD

**Discussion by mabianchet** [nglviewer/nglview#1012] [Q&A] [answered]
Hi 

With nglview.widget.shape.add_sphere(...)  an sphere can be added. Is any way to control its opacity/transparency?
Thanks

Mario

**Comment by hainm** [comment, marked as answer]
hi @mabianchet: Unfortunately no. Please see this thread to see if there is a work around for it: https://github.com/nglviewer/ngl/issues/376#issuecomment-333279149
